sources/updater/update_metadata/src/lib.rs

```diff
@@ -1,33 +1,43 @@
 #![warn(clippy::pedantic)]
 
 mod de;
+mod direction;
 pub mod error;
 mod se;
 
 use crate::error::Result;
 use chrono::{DateTime, Duration, Utc};
-use lazy_static::lazy_static;
+pub use direction::Direction;
 use parse_datetime::parse_offset;
 use rand::{thread_rng, Rng};
 use regex::Regex;
 use semver::Version;
 use serde::{Deserialize, Serialize};
 use snafu::{ensure, OptionExt, ResultExt};
+use std::cmp::Ordering;
 use std::collections::BTreeMap;
 use std::fs;
 use std::fs::File;
 use std::ops::Bound::{Excluded, Included};
 use std::path::Path;
-use std::str::FromStr;
-use tough;
+use tough::{self, Limits};
+
+#[macro_use]
+extern crate lazy_static;
```

Comment:
```suggestion
use lazy_static::lazy_static;
```

This syntax is deprecated, can we stick with the previous `use lazy_static::lazy_static;` statement up above?

Reply:
Addressed in [4f34544](https://github.com/bottlerocket-os/bottlerocket/compare/afc4b5d9ddfb7c6fb24f30abdaf6a5a000df222c..4f345446016e314c3da3f0855c92db5ca979f2a4)